### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by armgabrielyan**
This fixes small typos in the documentation.

### CODE REVIEW

File: docs/tutorials/features/graph_optimization.md [hunk 1]
```diff
@@ -13,13 +13,22 @@ ipex.enable_onednn_fusion(False)
...
 # Import the Intel Extension for PyTorch
 import intel_extension_for_pytorch as ipex
 
-model = models.__dict__["resnet50 "](pretrained=True)
+
+@dataclass
+class Args:
+    batch_size: int
+
+args = Args(batch_size=4)
```
Comment (jingxu10) [inline docs/tutorials/features/graph_optimization.md:29]: Seems like `args` is not used. Why adding it?

**Comment by jingxu10** [comment]
Thank for contributing to IPEX. Due to our current limitation of PR merging mechanism, I manually picked your code changes into the internal repo and synced out here. Please check the master branch with your code changes, and close this PR if there there are no further concerns. Thank you.

File: docs/tutorials/features/graph_optimization.md (hunk 1 above)
Comment (armgabrielyan) [reply docs/tutorials/features/graph_optimization.md:29]: It is used later in the code.

**Comment by armgabrielyan** [comment]
Thank you! The changes look good, closing this PR.

**Pull request closed without merging**